qemu/tests/ansible_test.py

```diff
@@ -41,10 +41,11 @@ def run(test, params, env):
     # Use this directory to copy some logs back from the guest
     test_harness_log_dir = test.logdir
 
-    vm = env.get_vm(params["main_vm"])
-    vm.verify_alive()
-    vm.wait_for_login()
-    guest_ip = vm.get_address()
+    guest_ip_list = []
+    for vm in env.get_all_vms():
+        vm.verify_alive()
+        vm.wait_for_login()
+        guest_ip_list.append(vm.wait_for_get_address(0, 240))
```

Comment:
The guest has already logged in, so `vm.get_address()` would be more suitable.

Reply:
Updated.